src/extension.ts

```diff
@@ -93,6 +93,42 @@ export function activate(context: ExtensionContext) {
         runTextInTerm(callableTerminal, wrappedText);
     }
 
+    async function runCommandWithSelectionOrWord(rCommand: string) {
+        const text = getWordOrSelection().join("\n");
+        const callableTerminal = await chooseTerminal();
+        let call = rCommand.replace("$$", text);
```

Comment:
`.replace()` seems only do it once. If user supply something like 

```json
{
        "description": "run print current word",
        "key": "ctrl+shift+1",
        "command": "r.runCommandWithSelectionOrWord",
        "when": "editorTextFocus",
        "args": "{ print($$); saveRDS($$, 'data/$$.rds'); }"
}
```

Only the first `$$` is replaced but user may probably expect that all `$$` should be replaced?

Reply:
Oh dear. Fixed.